content/posts/2022-09-26-bundle-option-in-crc.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Different bundle types option in crc"
+date: 2022-09-26T10:41:10+05:30
+---
+
+CRC provides a config option called `bundle` to configure the _location_ of the bundle, 
+until 2.9 release of CRC the bundle option could only able to accept local filesystem path
+which meant users needed to download bundle in advance before consuming it. The move to a
```

Comment:
missing a `the`, should be `ich meant users needed to download the bundle in advance...`